source/includes/extracts-server-status-projection-base.yaml

```diff
@@ -15,6 +15,16 @@ content: |
 ref: _serverStatus-output-changes
 content: |
 
+  - Starting in MongoDB 4.4.3, {{operationName}}:
+
+    - Updated field names from 'master' to 'primary' fields in
+    :serverstatus:`metrics.repl.network`. This change will require you
+    to update any code using the old field names. 
+    
+      - "notMasterLegacyUnacknowledgedWrites" -> "notPrimaryLegacyUnacknowledgedWrites"
+
+      - "notMasterUnacknowledgedWrites" -> "notPrimaryUnacknowledgedWrites" 
```

Comment:
Let's keep this simple:

> - Starting in MongoDB 4.4.3, {{operationName}}:
      -  Changed field name `notMasterLegacyUnacknowledgedWrites` to
          new field name `notPrimaryLegacyUnacknowledgedWrites`
      - Changed field name  `notMasterUnacknowledgedWrites` to
          new field name `notPrimaryUnacknowledgedWrites`

Reply:
And please link the new field names

Reply:
And please add this section to your other PR, in similar fashion. I notice it is missing there.

Reply:
I mentioned this in my note earlier - this section is only in one PR because it covers both changes rather than putting half in each of them. That confused the automated merge routine earlier.  The 2d reference will work properly if the other PR is merged first ( https://github.com/mongodb/docs/pull/4737 )